pkg/pureutils/pure_types.go

```diff
@@ -82,6 +83,12 @@ func GetPXPureSecret(namespace string) (PXPureSecret, error) {
 	return conf, err
 }
 
+func GetFASecretNotPresentInPureSecret(namespace string) (PXPureS3Secret, error) {
```

Comment:
Is this required?

Reply:
My bad got it mixed up. remove this function.